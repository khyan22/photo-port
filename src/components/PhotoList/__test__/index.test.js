import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import PhotoList from '..';

const currentCategory = {
  name: 'landscape'
};

afterEach(cleanup);

describe('PhotoList component', () => {
  it('renders', () => {
    render(<PhotoList category={currentCategory.name}/>)
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<PhotoList category={currentCategory.name} />);
    expect(asFragment()).toMatchSnapshot();
  });
});