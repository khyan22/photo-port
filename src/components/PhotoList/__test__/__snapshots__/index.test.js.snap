// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoList component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex-row"
    >
      <img
        alt="Green river"
        class="img-thumbnail mx-1"
        src="0.jpg"
      />
      <img
        alt="Docks"
        class="img-thumbnail mx-1"
        src="1.jpg"
      />
      <img
        alt="Panoramic village by sea"
        class="img-thumbnail mx-1"
        src="2.jpg"
      />
      <img
        alt="Domestic landscape"
        class="img-thumbnail mx-1"
        src="3.jpg"
      />
      <img
        alt="Park bench"
        class="img-thumbnail mx-1"
        src="4.jpg"
      />
    </div>
  </div>
</DocumentFragment>
`;
